### PR TITLE
clarify predict_trials targets

### LINKED ISSUE #566: predict_trials() returns ground truth labels from Dataset as trial_labels?
I thought `trial_labels` returned are computed from `trial_predictions`, which should be the same as what returned by calling `predict()`. But it returns the ground truth labels from the dataset. Is this is intentional?

code from `.training.scoring.predict_trials`

```python
loader = DataLoader(
    dataset=dataset,
    batch_size=batch_size,
    shuffle=False,
    num_workers=num_workers,
)
device = next(module.parameters()).device
all_preds, all_ys, all_inds = [], [], []
with torch.no_grad():
    for X, y, ind in loader:
        X = X.to(device)
        preds = module(X)
        all_preds.extend(preds.cpu().numpy().astype(np.float32))
        all_ys.extend(y.cpu().numpy().astype(np.float32))
        all_inds.extend(ind)
```

### PR DESCRIPTION
**Pull request by tayal-sarthak**
Closes #566
 
Summary

`predict_trials` returns model predictions alongside ground truth targets taken directly from the dataset. The existing docstrings described the second value as `trial_labels`, which could be interpreted as labels derived from model predictions.

This update renames the documented return value to `trial_targets`. It also states that these values are ground truth dataset targets. Runtime behavior remains unchanged.

### CODE REVIEW

**Comment by tayal-sarthak** [comment]
let me know your thoughts!! :)

**Review by bruAristimunha — APPROVED** [review]

**Comment by bruAristimunha** [comment]
supper good! @tayal-sarthak :)
